scipy/special/_basic.py

```diff
@@ -3151,3 +3152,36 @@ def zeta(x, q=None, out=None):
         return _ufuncs._riemann_zeta(x, out)
     else:
         return _ufuncs._zeta(x, q, out)
+
+      
+def softplus(x, **kwargs):
+    r"""
+    Compute the softplus function element-wise.
+
+    The softplus function is defined as: ``softplus(x) = log(1 + exp(x))``.
+    It is a smooth approximation of the rectifier function (ReLU).
+
+    Parameters
+    ----------
+    x : array_like
+        Input array-like object.
+
+    **kwargs :
+        Additional keyword arguments to be passed to `np.logaddexp`.
```

Comment:
I think this line would be confusing to users. It might be easier to understand if the possible keyword arguments were explicitly written, without reference to `np.logaddexp`.